storage/innobase/xtrabackup/doc/source/installation/docker.rst

```diff
@@ -1,46 +1,102 @@
 .. _docker:
 
-Running Percona XtraBackup in a `Docker` container
+Run *Percona XtraBackup* in a `Docker` container
 ********************************************************************************
 
 You may run Percona XtraBackup in a `Docker` container without
-having to install it. All required libraries come already installed in
+having to install it. All required libraries are installed in
 the container.
 
 Being a lightweight execution environment, `Docker` containers enable creating
-configurations where each program runs in a separate container. You may run
+configurations where each program runs in a separate container. You can run
 Percona Server for MySQL in one container and Percona XtraBackup in another.
 
-You create a new `Docker` container based on a `Docker` image, which works as a
-template for newly created containers. `Docker` images for Percona XtraBackup
-are hosted publicly on Docker Hub `here <https://hub.docker.com/r/percona/percona-xtrabackup>`__.
+A `Docker` container is based on a `Docker` image, which works as a
+template for newly created containers. The `Docker` images for Percona XtraBackup are hosted publicly on Docker Hub `here <https://hub.docker.com/r/percona/percona-xtrabackup>`__.
```

Comment:
I suggest making the Docker Hub the link and get rid of "here" placeholder.

Reply:
Fixed.